Recruitment/Exercises/js_engineer_contract.md

```diff
@@ -0,0 +1,35 @@
+## Requirements
+
+Please build a React app displaying the 5 day weather forecast for a location of your choice.
+
+1. Do not spend more than 4 hours on your solution **(even if you do not complete the entire exercise)**.
+2. Use **React** to implement the solution.
```

Comment:
As on #42 I'd simply suggest React or any virtual dom lib as the principle stays the same